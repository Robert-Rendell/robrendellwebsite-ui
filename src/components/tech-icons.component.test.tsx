import React from "react";
import { render } from "@testing-library/react";
import { TechIconsComponent } from "./tech-icons.component";

describe("TechIconsComponent", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<TechIconsComponent/>);
    expect(asFragment()).toMatchSnapshot();
  });
});