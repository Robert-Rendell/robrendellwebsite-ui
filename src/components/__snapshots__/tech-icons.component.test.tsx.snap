// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechIconsComponent should match snapshot 1`] = `
<DocumentFragment>
  <img
    alt="React"
    height="40"
    src="logo.svg"
    width="40"
  />
  <img
    alt="TypeScript"
    class="tech-icons"
    height="30"
    src="ts-logo.png"
    width="30"
  />
  <img
    alt="nodeJs"
    class="tech-icons nodejs-icon"
    height="30"
    src="nodejs-logo.png"
    width="30"
  />
  <span
    class="tech-icons aws-icon-bg"
  >
    <img
      alt="nodeJs"
      class="aws-icon"
      height="20"
      src="aws-logo.png"
      width="30"
    />
  </span>
</DocumentFragment>
`;
